yarn.lock

```diff
@@ -1468,6 +1468,33 @@
   resolved "https://registry.yarnpkg.com/@spectrum-css/closebutton/-/closebutton-1.2.13.tgz#a02a33e211349138bcde0988cd4317749a7618e4"
   integrity sha512-V/P/XVWPh/TR2Qo66vn6o+vi5/utDbrMIdNV+zuF1dbl1wxH5QMOdCQrQOtzHrzcwulJRu3+Sk759VRPojLYCA==
 
+"@spectrum-css/component-builder-simple@^1.0.0-beta.0":
```

Comment:
hmm, did we have major work in the component builder. Still  on  1.0.0.

Reply:
Good catch, looks like I had this wrong in my package.json file. I changed line 23 of package.json to
    "@spectrum-css/component-builder-simple": "^2.0.0",